tests/integration/helpers.test.js

```diff
@@ -58,13 +58,14 @@ describe('GET /helpers', () => {
       { name: 'coach', expectedCode: 200 },
```

Comment:
il faudrait rajouter une vérif au test should return list of helpers pour vérifier que la route renvoie bien la liste

Reply:
done